tensorflow/python/keras/engine/base_layer_test.py

```diff
@@ -630,6 +630,11 @@ def call(self, x, y=None):
     out = self.evaluate(layer(x=x, y=y))
     self.assertAllClose(out, 2 * np.ones((10, 1)))
 
+  def test_relu_layer_with_dtype(self):
```

Comment:
Thanks for the change, can you move this to the backend_test.py under test_relu()